### ISSUE TITLE
Wildcard IP not support

### ISSUE THREAD

**Issue by amiralkizaru** [tomnomnom/httprobe#26]
### Definition
Hello Tomnomnom

Thank you for the tool I wanted check port `80` and port `443` of some subdomain that has this form
`ipaddress.*.*.organisation.com`
example
`1046170.ostk.bm2.prod.gq1.yahoo.com` or `105.240.139.98.aprez.ostk.prod.bf1.yahoo.com`
but apparently httprobe doesn't like this form

Regards

**Comment by adsadfasdfas** [comment]
Yeah you're right. It seems like httprobe can't take care subdomain with number such as 160.555.473.588.att.com

**Comment by ManShum815** [comment]
Is anyone has a solution to this?
